### PR TITLE
Complexity squash below C

### PR DESCRIPTION
**Pull request by CoMPaTech**
Though complexity is now squashed, not really fond of how it looks now, lets just PR to share thoughts.

### CODE REVIEW

File: plugwise/helper.py [hunk 1]
```diff
@@ -175,103 +211,112 @@ async def request(
...
+                "name": loc.name,
+                "types": loc.types,
+                "members": loc.members,
             }
 
-        self._home_location = home_location
-
         return
 
+    def appliance_stretch_class_finder(self, appliance, appl):
```
Comment (bouwew) [inline plugwise/helper.py:296]: The function-name is too specific, the 1st part is covering the devices connected to a Stretch, at the moment we are only supporting Circles and Stealths, the 2nd part is covering Plugs connected to an Adam.

I would suggest 'energy_device_info_finder' or something similar.

File: plugwise/helper.py [hunk 2]
```diff
@@ -175,103 +211,112 @@ async def request(
...
+        if self.smile_type != "stretch" and "plug" in appl.types:
+            locator = ".//logs/point_log/electricity_point_meter"
+            mod_type = "electricity_point_meter"
+            module_data = self.get_module_data(appliance, locator, mod_type)
+            appl.v_name = module_data[0]
+            appl.model = version_to_model(module_data[1])
+            appl.fw = module_data[3]
+            return appl
+
     def appliance_class_finder(self, appliance, appl):
```
Comment (bouwew) [inline plugwise/helper.py:320]: Please change the name to `appliance_info_finder`, the use of `class` in the name does not cover its purpose.

**Comment by bouwew** [comment]
Looking good in general!
Indeed, the complexity has become less but the structure has become less clear. 
On the other hand, we'll get used to that :)

**Comment by bouwew** [comment]
One more thing:
```
    def appliance_class_finder(self, appliance, appl):
        """Determine class per appliance."""
```

Please change to something like:
```
    def appliance_info_finder(self, appliance, appl):
        """Collect device info: firmware, model and vendor name."""
```

File: plugwise/helper.py (hunk 1 above)
Comment (CoMPaTech) [reply plugwise/helper.py]: Good call!

**Comment by CoMPaTech** [comment]
Apparently rebased shizzled the legacy around and it ended back in it's old position (increasing complexity). Bumped alpha to 6

**Pull request merged**